napari/layers/points/points.py

```diff
@@ -447,7 +450,7 @@ def __init__(
 
         self.experimental_canvas_size_limits = experimental_canvas_size_limits
         self.shading = shading
-        self._antialias = True
+        self.antialiasing = 1 if antialiasing is None else antialiasing
```

Comment:
Why this if clause, and why 1 if None? I would have just put 1 as default and have no clause here, but maybe I'm missing something? Or at least 0 if None...

Reply:
I was following the pattern of `None` being the default kwarg for most things in the constructor

Reply:
Not sure why that's the case, actually; I would say it makes sense only in a few cases (such as with complicated defaults like affines, mutable things like metadata, or autogenerated defaults like `name`), but I don't see why it should be the case here (or in stuff like `n_dimensional`, for example).